### PR TITLE
Adding requirements.yml

### PR DESCRIPTION
**Pull request by idhaoui**

### CODE REVIEW

File: deploy-container.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 - name: deploy httpd container
   hosts: all
-  become: true
```
Comment (cbolz) [inline deploy-container.yml]: why do you remove this? copy into webroot should require root privileges, no?

**Comment by idhaoui** [comment]
Priv escalation is done on the job template.

**Comment by cbolz** [comment]
> 

I personally prefer it to be in the playbook because it makes it easier to use the same playbook on the CLI without changes or additional command line switches - never really saw a benefit in setting this in the JT

**Comment by idhaoui** [comment]
I am neutral to that, both work for me. I  will restore it again. I will
update the documentation accordingly as it mentions that you should tick
the priv escalation.

**Pull request merged**